Makefile

```diff
@@ -1,3 +1,5 @@
+DOCKER_BUILD ?= docker build --force-rm
```

Comment:
Let's avoid mentioning docker, let's use CONTAINER instaead, and podman as build system

Reply:
Ack, will `s/docker/CONTAINER/g`